assets/section-main-product.css

```diff
@@ -27,23 +27,35 @@
   }
 
   .product--thumbnail .thumbnail-list {
-    padding-right: var(--media-shadow-horizontal-offset);
+    padding-inline-end: var(--media-shadow-horizontal-offset);
   }
 
   .product__info-wrapper {
-    padding: 0 0 0 5rem;
+    padding-top: 0;
+    padding-bottom: 0;
+    padding-inline-start: 5rem;
+    padding-inline-end: 0;
```

Comment:
Yeah my thought would potentially start changing our language in the editor. But we would need to find the right language that isn't confusing. Otherwise like you said it might be painful to adapt to the theme settings and not be able to use the flexibility of logical properties.